usaspending_api/api_contracts/contracts/v2/search/spending_by_award.md

```diff
@@ -295,3 +295,6 @@ List of filterable award types
 - `IDV_C`
 - `IDV_D`
 - `IDV_E`
+
+## DEFC (enum[string])
+List of Disaster Emergency Fund (DEF) Codes (DEFC) defined by legislation at the time of writing
```

Comment:
all of it... https://github.com/fedspendingtransparency/usaspending-api/blob/dev/usaspending_api/api_contracts/contracts/v2/disaster/recipient/count.md#members

Reply:
do i look like i know how to read.

anyway should be fixed now